Intacct/BatchToJournal.ascx

```diff
@@ -2,6 +2,7 @@
 
 <asp:UpdatePanel ID="upnlSync" runat="server">
     <ContentTemplate>
+        <Rock:ModalAlert ID="maWarningDialog" runat="server" />
         <div class="row">
```

Comment:
```suggestion
        <div class="row">
            <Rock:ModalAlert ID="maWarningDialog" runat="server" />
```
I believe we should put the modal inside the row div to keep its display confined with the rest of the content of the block.

Reply:
I'll adjust it accordingly...